FileShareService.DesktopClient.Core/JobsParser.cs

```diff
@@ -12,10 +15,16 @@ public interface IJobsParser
 
     public class JobsParser : IJobsParser
     {
+        private static readonly List<string> ValidJobActions =
+            new List<string>() { "newBatch", "appendAcl", "setExpiryDate" };
+
+        private List<string> jobIdCollection;
```

Comment:
Please add an initial value for `jobIdCollection` to its declaration, to avoid the compiler warning.
````
private List<string> jobIdCollection = new List<string>();
````

and remove the following assignment from `Parse()`
````
jobIdCollection = new List<string>();
````

Reply:
Fixed